tests/foreman/cli/test_capsule.py

```diff
@@ -47,7 +47,8 @@ def test_negative_create_with_url(self):
         @Assert: Proxy is not created
         """
         # Create a random proxy
-        with self.assertRaises(CLIFactoryError):
+        with self.assertRaisesRegex(CLIFactoryError,
+                                    u'Could not create the proxy:'):
```

Comment:
Please use one of following styles instead:
```python
        with self.assertRaisesRegex(
                CLIFactoryError, u'Could not create the proxy:'):
```
or 
```python
        with self.assertRaisesRegex(
            CLIFactoryError,
            u'Could not create the proxy:',
        ):
```
We're trying to avoid such half-line-long indentations.